packages/@aws-cdk/cdk/lib/app.ts

```diff
@@ -30,42 +108,27 @@ export class App extends Root {
   }
 
   /**
-   * Runs the program. Output is written to output directory as specified in the request.
+   * @deprecated
```

Comment:
`@deprecated` should mention why & what users should do instead (even if that is "nothing").